apps/devhub/tests/test_views.py

```diff
@@ -2539,9 +2539,9 @@ def xpi(self):
                     'rb')
 
     @mock.patch.object(waffle, 'flag_is_active')
+    @mock.patch('devhub.tasks.validate')
```

Comment:
(Also missing, clearly, is a bunch of comments about all of the above, which I'll add)